readthedocs/rtd_tests/tests/test_doc_builder.py

```diff
@@ -281,7 +254,7 @@ def test_get_theme_name_with_feature_flag(self, checkout_path, run):
 
     @patch('readthedocs.doc_builder.base.BaseBuilder.run')
     @patch('readthedocs.projects.models.Project.checkout_path')
-    def test_append_conf_create_yaml(self, checkout_path, run):
+    def test_append_conf_does_not_create_yaml(self, checkout_path, run):
```

Comment:
This should be named something like `test_mkdocs_yaml_not_found`

Reply:
Okay, thanks :)